src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -205,15 +205,15 @@ private void initdb()
         List<String> command = Lists.newArrayList(pgBin("initdb"), "-A", "trust", "-U", PG_SUPERUSER,
             "-D", dataDirectory.getPath(), "-E", "UTF-8");
         command.addAll(createLocaleOptions());
-        system(errorRedirector, outputRedirector, command.toArray(new String[command.size()]));
+        system(command.toArray(new String[command.size()]));
```

Comment:
? I guess the code will show later why these params dropped

Reply:
I got rid of the method that allowed you customize the output and error redirection.  When calling `system`, we just _always_ use `PIPE` now.

Reply:
(it did)

Reply:
: ]